projects/ad4630_fmc/common/ad463x_adaq42xx_bd.tcl

```diff
@@ -21,18 +22,22 @@ set cnv_ref_clk 100
 # NOTE: this is a default value, software may or may not change this
 set adc_sampling_rate 1000000
 
-#create_bd_intf_port -mode Master -vlnv analog.com:interface:spi_master_rtl:1.0 ad463x_spi
+# specify the MAX17687 and LT8608 SYNC signal frequency (400KHz)
+set max17687_sync_freq 400000
 
-create_bd_port -dir O ad463x_spi_sclk
-create_bd_port -dir O ad463x_spi_cs
-create_bd_port -dir O ad463x_spi_sdo
-create_bd_port -dir I -from [expr $NUM_OF_SDI-1] -to 0 ad463x_spi_sdi
+#create_bd_intf_port -mode Master -vlnv analog.com:interface:spi_master_rtl:1.0 ad463x_adaq42xx_spi
 
-create_bd_port -dir I ad463x_echo_sclk
+create_bd_port -dir O ad463x_adaq42xx_spi_sclk
+create_bd_port -dir O ad463x_adaq42xx_spi_cs
+create_bd_port -dir O ad463x_adaq42xx_spi_sdo
+create_bd_port -dir I -from [expr $NUM_OF_SDI-1] -to 0 ad463x_adaq42xx_spi_sdi
 
-create_bd_port -dir I ad463x_busy
-create_bd_port -dir O ad463x_cnv
-create_bd_port -dir I ad463x_ext_clk
+create_bd_port -dir I ad463x_adaq42xx_echo_sclk
+create_bd_port -dir I ad463x_adaq42xx_busy
+create_bd_port -dir O ad463x_adaq42xx_cnv
+create_bd_port -dir I ad463x_adaq42xx_ext_clk
+
+create_bd_port -dir O max17687_sync_clk
```

Comment:
`max17687_sync_clk` logic is generated even for `AD463X_ADAQ42XX_N=1`, is this ok?
Same for other max17687 related logic.

Reply:
that is the intend, will have the sync clock generated also for ad4630, but would not be used